src/main/java/graphql/annotations/processor/retrievers/GraphQLFieldRetriever.java

```diff
@@ -196,7 +210,7 @@ private TypeFunction getTypeFunction(Field field, ProcessingElementsContainer co
         return typeFunction;
     }
 
-    private GraphQLOutputType getGraphQLConnection(AccessibleObject field, GraphQLOutputType type, Relay relay, Map<String, graphql.schema.GraphQLType> typeRegistry) {
+    private GraphQLOutputType getGraphQLConnection(AccessibleObject field, graphql.schema.GraphQLType type, Relay relay, Map<String, graphql.schema.GraphQLType> typeRegistry) {
```

Comment:
delete the "graphql.schema", use import instead

Reply:
This class uses two GraphQLType - one of the annotations project and one of graphql java. However I changed the import to graphql.schema and put the prefix when using the annotations' GraphQLType.